aiogoogle/__init__.py

```diff
@@ -20,3 +20,10 @@
     HTTPError,
     ValidationError,
 )
+from . import utils  # noqa: F401  imported but unused
+from . import validate  # noqa: F401  imported but unused
+
+
+__all__.extend(client.__all__)
```

Comment:
This failed in the tests because some modules (client, resource, excs) are not defined.
I think we have 2 options here:

```python
from .client import Aiogoogle, __all__ as client__all__  # so it doesn't conflict with the other `__all__`s
```

or 

```python
from . import client
```

First option might be better because we won't be importing the entire module (which probably won't matter from a performance perspective, but might matter from an API design perspective. This way the entire module isn't import-able). Not sure about the naming conventions here though :) feel free to improvise.

Reply:
Importing `__all__`  would also trigger warning as string `"__all__"` itself is not included in `__all__` - which we can also safely ignore for the convenience as such minor warning wouldn't affect the user in any case :)

One confusing thing is: I am not sure whether using `from .<submodule_name> import ~~~` also import the submodule itself or not.

From that previous asyncio's `__init__.py`:
```python
from .base_events import *
from .coroutines import *
from .events import *
from .exceptions import *
...

__all__ = (base_events.__all__ +
           coroutines.__all__ +
           events.__all__ +
           exceptions.__all__ +
           ...
```

Here they imported using same format (except that it's star import), yet the module `asyncio` works flawlessly.

I am not sure if these is possible when imported using star or it is difference per python versions, do you know anything about it?

Reply:
> Importing __all__ would also trigger warning as string "__all__" itself is not included in __all__ - which we can also safely ignore for the convenience as such minor warning wouldn't affect the user in any case :)

Agreed

About the second issue, I have no idea whether the asterisk is what's causing this to not fail. I'm happy to go with whatever approach you think is best. Just as long as all the CI jobs are passing :)

I previously thought that the error happening in this PR was in the tests job and not the lint job. Would've just asked you to add a comment that ignores this error so that the lint CI job doesn't fail.